packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -393,6 +393,17 @@ describeWithDOM('mount', () => {
       );
       expect(wrapper.find('input.foo').length).to.equal(1);
     });
+    
+    it('should strip out any non-hostNode', () => {
+      const Foo = props => <div {...props} />;
+      const wrapper = mount(
+        <div>
+          <Foo className="foo" />
+        </div>,
+      );
+      expect(wrapper.find('.foo').length).to.equal(2);
+      expect(wrapper.find('.foo').hostNodes().length).to.equal(1);
```

Comment:
could we assert on the contents too, not just the length? something like:
```js
const hostNodes = wrapper.find('.foo').hostNodes();
expect(hostNodes).to.have.lengthOf(1);
expect(hostNodes.is('div')).to.equal(true);
```

Reply:
😩 I was trying to perform the PR trough the GitHub UI but I guess I'll have to clone the project locally 😅

Reply:
done!